### PR TITLE
Added a setupapi.dev.log files sample

### PR DESCRIPTION
**Pull request by joinimran**
As per request, I have added a setupapi.dev.log file sample.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4043

### CODE REVIEW

**Comment by jvsam** [comment]
@joinimran I sent you my feedback. Thank you.

**Comment by illfated** [comment]
@joinimran : I noticed you have selected to use the MarkDown code block syntax highlight keyword `cpp` (a.k.a. C++ ),  is there a precedent for this use with that example log output? (I would have thought it does not need a code syntax keyword, or maybe something closer to flat text output. Or -- were you possibly thinking about a different meaning for cpp ?

**Comment by illfated** [comment]
Very well. Thank you for the good feedback and additional information.

**Comment by joinimran** [comment]
@jvsam This PR has been updated as per your suggestion. To make this PR workable, please get PR https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5322 approved and merged and I have added the code section in the log file doc.

**Comment by jvsam** [comment]
Thank you @joinimran!

@nenonix @JohanFreelancer9 for copy review. Thank you.

**Comment by illfated** [comment]
👍 Good solution, your PR #5322 (**Added a log file for setupapi.dev.log**) is now merged and the result looks quite good. I also like how this PR was changed in accordance with the other one.

Looking forward to the copy review, as the added sentence could use some minor adjustments for improved language fluency.

File: windows/deployment/upgrade/resolution-procedures.md [hunk 1]
```diff
@@ -32,11 +33,10 @@ A frequently observed result code is 0xC1900101. This result code can be thrown
 - Event logs: $Windows.~bt\Sources\Rollback\*.evtx
 - The device install log: $Windows.~bt\Sources\Rollback\setupapi\setupapi.dev.log
 
-The device install log is particularly helpful if rollback occurs during the sysprep operation (extend code 0x30018).  To resolve a rollback due to driver conflicts, try running setup using a minimal set of drivers and startup programs by performing a [clean boot](https://support.microsoft.com/kb/929135) before initiating the upgrade process.
+The device install log is particularly helpful if rollback occurs during the sysprep operation (extend code 0x30018).  To resolve a rollback due to driver conflicts, try running setup using a minimal set of drivers and startup programs by performing a [clean boot](https://support.microsoft.com/kb/929135) before initiating the upgrade process. You can find the log file segment for error code at [Log files](https://docs.microsoft.com/windows/deployment/upgrade/log-files#analyze-log-files).
```
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/resolution-procedures.md]:
```suggestion
The device install log is particularly helpful if rollback occurs during the sysprep operation (extend code 0x30018).  To resolve a rollback due to driver conflicts, try running setup using a minimal set of drivers and startup programs by performing a [clean boot](https://support.microsoft.com/kb/929135) before initiating the upgrade process. For more details on the log file segment of the error code, see [Log files](https://docs.microsoft.com/windows/deployment/upgrade/log-files#analyze-log-files).
```

**Comment by joinimran** [comment]
Thanks for your feedback @JohanFreelancer9. This totally makes sense. 

@jvsam , Ready to go live.

**Comment by jvsam** [comment]
@greg-lindsay thank you for merging #5322. We can now add a link to the log file. Kindly review this PR and merge if there are no issues. Regards!

**Comment by jvsam** [comment]
@officedocsbot assign @greg-lindsay

**Comment by jvsam** [comment]
Hi again @greg-lindsay, just a friendly follow-up. Hoping you'll be able to review this pull request soon. Thank you.

**Comment by jvsam** [comment]
Hi again @greg-lindsay, just a friendly follow-up on the status of this pull request. This PR will add a link to the *setupapi.dev.log* content, which has been previously approved. Thank you

**Comment by jvsam** [comment]
Hi @greg-lindsay, hope all is well. Just following up on this PR. If you think adding a reference link to the Log File content page is not appropriate in context, kindly let us know. Thank you again for your assistance.

**Comment by illfated** [comment]
#sign-off

**Comment by jvsam** [comment]
Hi @greg-lindsay, just following up on the escalation. Since there are already multiple links that redirect users to the [Log Files](https://docs.microsoft.com/windows/deployment/upgrade/log-files) content page, please let us know if the additional link is not necessary so we can just close this PR. Your guidance is appreciated, thank you.

FYI @joinimran

**Comment by joinimran** [comment]
Closing this PR as there is no response from the author and info has already been updated as per users' request.

**Pull request closed without merging**